src/main/resources/view/HelpWindow.css

```diff
@@ -17,3 +17,7 @@
 #helpMessageContainer {
     -fx-background-color: derive(#1d1d1d, 20%);
 }
+
+#helpMessageScrollPane {
+    -fx-max-height: 600
+}
```

Comment:
```css
#helpMessageContainer {
    -fx-background-color: derive(#1d1d1d, 20%);
    -fx-min-width: 1080;
}

#helpMessageScrollPane {
    -fx-max-height: 600;
    -fx-min-width: 1100;
}
```

this seems to work for me, 1080 is calculated by taking 1100-20

Reply:
Thanks for the help, the window looks nice